### PR TITLE
Adjusting README headers

### PR DESCRIPTION
**Pull request by taleldayekh**

### CODE REVIEW

**Comment by christianbrb** [comment]
@taleldayekh If I see it, I would like to have it different ;) If you don't mind, I'll implement the changes on my own within this PR, ok?

**Comment by taleldayekh** [comment]
> @taleldayekh If I see it, I would like to have it different ;) If you don't mind, I'll implement the changes on my own within this PR, ok?

Feel free to do so :)

**Comment by christianbrb** [comment]
Done :) I have assigned @nephix for a review

File: README.md [hunk 1]
```diff
@@ -221,6 +218,7 @@ Structure of the state file:
 }
 ```
 
+### Additional options for minting and target
```
Comment (nephix) [inline README.md:221]: I guess it should be capitalized, like the other headlines
Also the wording here is a bit odd: `Additional options for ... target`
Maybe makes sense to make `Minting Manually` and `Use a Raiden Full Node` a 3rd level header

Comment (christianbrb) [reply README.md:221]: @nephix Thanks for your feedback

Looking at it again I would just remove the two sections (Minting manually and use a Raiden Full Node) as it is our main readme and I want to keep it short and precise. @taleldayekh Are you ok with it?

Comment (christianbrb) [reply README.md:221]: Talked with @taleldayekh and we agreed to remove it. I am implementing this now.

**Pull request merged**